packages/graphql-language-server/package.json

```diff
@@ -0,0 +1,80 @@
+{
+  "name": "graphql-language-server",
```

Comment:
do we have a conflict here?  https://www.npmjs.com/package/graphql-language-server

Reply:
Yes. Changed to `stardog-graphql-language-server`.